scipy/optimize/_linprog.py

```diff
@@ -338,13 +344,30 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     This section describes the available solvers that can be selected by the
     'method' parameter.
 
+    :ref:`'highs-simplex' <optimize.linprog-highs-simplex>` and
+    :ref:`'highs-ipm' <optimize.linprog-highs-ipm>` are interfaces to the
+    HiGHS simplex and interior-point method solvers [13]_, respectively.
+    'highs' chooses between the two automatically. These are the fastest linear
+    programming solvers in SciPy, especially for large, sparse problems.
     :ref:`'interior-point' <optimize.linprog-interior-point>` is the default
-    as it is typically the fastest and most robust method.
+    as it was the fastest and most robust method before the recent
+    addition of the HiGHS solvers. This may change in a future release.
```

Comment:
Please remove _"This may change in a future release"_. That'd be a backwards compat break, so seems unlikely to happen in a minor release at least, and you shouldn't make users think about that.

Reply:
Presumably HiGHS will be the default solver in due course. Note that in a future release of HiGHS there will be the option of allowing it to decide whether to use interior point (IPM) or simplex to solve an LP based on problem characteristics and what starting solution is available. In addition, as with Gurobi, where multiple cores are available HiGHS may run IPM and simplex on separate cores, until one of them has found a solution.

Reply:
OK. While the default will likely change to a HiGHS solver, I will not warn users ahead of time.